doc/nrf/libraries/modem/modem_slm.rst

```diff
@@ -31,6 +32,7 @@ The library is enabled and configured entirely using the Kconfig system.
 Configure the following Kconfig options to enable this library:
 
 * :kconfig:option:`CONFIG_MODEM_SLM` - Enables the Modem SLM library.
+* :kconfig:option:`CONFIG_MODEM_SLM_DMA_MAXCNT` - Configures UART RX EasyDMA buffer size, which is configured to 1024 bytes by default.
```

Comment:
nit: MAXCNT looks more like `UART_RX_BUF_NUM`. I'd prefer `MAXLEN` . But it's ok to remain.

Reply:
It's named after UARTE register RXD.MAXCNT, but we could change to MAXLEN as suggested